fuzz/src/invoice_request_deser.rs

```diff
@@ -11,7 +11,7 @@ use crate::utils::test_logger;
 use bitcoin::secp256k1::{self, Keypair, Parity, PublicKey, Secp256k1, SecretKey};
 use core::convert::TryFrom;
 use lightning::blinded_path::payment::{
-	BlindedPaymentPath, Bolt12OfferContext, ForwardNode, ForwardTlvs, PaymentConstraints,
+	BlindedPaymentPath, Bolt12OfferContext, PaymentForwardNode, ForwardTlvs, PaymentConstraints,
```

Comment:
I think rustfmt will fail over this. On the first push a few other of these imports were out of order too